sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -18,6 +18,35 @@ Use the client library for App Configuration to create and manage application co
 
 ### Adding the package to your product
 
+#### Managed Dependencies
+
+Java SDKs for Azure releases a BOM (Bill of Materials) which deals with inter-artifact dependencies and is the recommended way to consume artifacts. The BOM contains a set of artifacts that have been verified to share a common set of dependencies. The following is how to include App Configuration in your product when using the BOM.
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-sdk-bom</artifactId>
+            <version>1.0.0-beta.1</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+
+<dependencies>
+    <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-data-appconfiguration</artifactId>
+    </dependency>
+</dependencies>
+```
+
+#### Individual Dependency
```

Comment:
Wonder if this section is necessary. We should encourage users to use the BOM and for individual dependency we can just point them to maven central (have a link here) and ask to copy the latest version if they choose to. This will get us out of managing versions in readme and having an incorrect version for short duration of time from when the readme is updated to when the package is released to maven.

Reply:
So, only have the Managed Dependencies section (probably remove the title) and after it have a link that says `If you wish to use individual references find the latest version here[link to maven]`?